src/toil/wdl/wdl_synthesis.py

```diff
@@ -665,6 +666,9 @@ def run(self, fileStore):
                                  fileStore.logToMaster("{jobname}")
                                  tempDir = fileStore.getLocalTempDir()
                                  
+                                 b3a2e464d6a4464cb138952a7d042595_stdout_file = os.path.join(tempDir, 'stdout')
```

Comment:
We should document that this variable is prefixed with a high-entropy string to try and make it unique, so it can't collide with anything else in the genrated code.

Are you sure that's the right approach, rather than something human-readable, underscore-rich, and Toil-y as a prefix, like `_toil_wdl_internal__stdout_file`? It's not statistically guaranteed to avoid a collision unless someone actually copy-pastes it, but it is more meaningful if someone has to go spelunking through the generated code.

Reply:
If we are going to use a high-entropy string, rather than copy-paste it everywhere, we should source it from one place in the generator.

Reply:
Good point! A `_toil_wdl_internal__` prefix is definitely more readable than a high-entropy string. I'll switch to that instead. I wonder if I should also change [`asldijoiu23r8u34q89fho934t8u34fcurrentworkingdir`](https://github.com/DataBiosphere/toil/blob/releases/4.2.x/src/toil/wdl/wdl_synthesis.py#L115) since I based the naming on that?

Reply:
I would change that too.